### DISCUSSION TITLE
Matched snRNAseq/Visium

### DISCUSSION THREAD

**Discussion by edward130603** [BayraktarLab/cell2location#176] [General] [answered]
Thanks for developing this tool and providing data to the community. I was interested in using the mouse brain dataset provided in your paper to see how the results change depending on whether matched snRNAseq or other reference is used. Could you provide the details on which snRNAseq samples (e.g. 5705STDY8058280) are matched with which Visium samples (e.g. ST8059048). Thanks!

**Comment by vitkl** [comment, marked as answer]
Hi @edward130603 

I would say that all of the samples from one animal perfectly matched. Two animals are of different sex so there are some txn mismatches. Still, this is not as different as providing data for an incorrect region or completely different tissue.

Othewise the sample pairing is as follows:
5705STDY8058280 ST8059048
5705STDY8058281 ST8059049
5705STDY8058282 ST8059050

5705STDY8058283 ST8059051
5705STDY8058284 ST8059052
5705STDY8058285
